README.md

```diff
@@ -279,9 +279,15 @@ RMB implementing modules expect a set of environment variables to be passed in a
  - DB_QUERYTIMEOUT
  - DB_CHARSET
  - DB_MAXPOOLSIZE
+ - RMB_EXPLAIN_QUERY_THRESHOLD
 
 Environment variables with periods/dots in their names are deprecated in RMB because a period is [not POSIX compliant](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html) and therefore some shells, notably, the BusyBox /bin/sh included in Alpine Linux, strip them (reference: [warning in OpenJDK docs](https://hub.docker.com/_/openjdk/)).
 
+The environment variable `RMB_EXPLAIN_QUERY_THRESHOLD` is not observed by Postgres itself, but
+is a value - in milliseconds - that triggers query execution analysis. If a single query exceeds this
+threshold, it will be analyzed by using `EXPLAIN ANALYZE`. Note that this in turn further adds time
+to the query, so this should only be executed for slow queries that needs further attention. Like the DB-environment variables this pertains per RMB-module (process).
```

Comment:
This tells me: "this should only be executed for slow queries" How can I enable this per query if this is per RMB-module?
Or should it say: "so this should only be executed in a separate performance test environment."?

Reply:
No. We want it to be enabled by default with NO modification to any environment so we get information about unacceptable slow queries.

Reply:
Thank you, can you add to README that it is enabled by default and the default threshold value?